stackslib/src/chainstate/burn/db/sortdb.rs

```diff
@@ -357,6 +358,9 @@ impl FromRow<StackStxOp> for StackStxOp {
         let stacked_ustx = u128::from_str_radix(&stacked_ustx_str, 10)
             .expect("CORRUPTION: bad u128 written to sortdb");
         let num_cycles = row.get_unwrap("num_cycles");
+        let signer_key_str: String = row.get_unwrap("signer_key");
+        let signer_key = serde_json::from_str(&signer_key_str)
+            .expect("CORRUPTION: DB stored bad transition ops");
```

Comment:
This works because we have `DEFAULT NULL` in the DB migration, right? Just want to double check

Reply:
I think so...along with the default de/serialization. I had this doubt for [this](https://github.com/stacks-network/stacks-core/pull/4412/files#diff-f6160bafb8ea4919ebd27f68cd1a9b56e66c8e0be60c544cc379b6aa8e2c960cR5308) too but I included None for signer_key in unit tests and it doesn't cause panic.

Reply:
If `NULL` is allowed, then shouldn't this be:

```
let signing_key_str_opt: Option<String> = row.get_unwrap("signer_key");
```

Reply:
Both work but I will use your suggestion with match statement since it'll be more clear/safer.
When I debugged mine with signer_key as `None`, it evaluated to:

`let signer_key_str: String = row.get_unwrap("signer_key");`
-> "null"
 `let signer_key = serde_json::from_str(&signer_key_str)
            .expect("CORRUPTION: DB stored bad transition ops");`
->  None